src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/AvoidInlineConditionalsCheck.java

```diff
@@ -42,6 +42,16 @@
  * <pre>
  * &lt;module name=&quot;AvoidInlineConditionals&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int x = 5;
+ * boolean result = (x == 5) ? true : false; // violation
+ * boolean foo = "foo".equals("bar") ? (x == 5) ? true : false : true; // violation
+ * boolean foobar = (x == 5); // OK
```

Comment:
@Gaurav-Punjabi I think the right replacement for the ternary operator from the Check description is
```(java)
String b;
if (a == null || a.length() < 1) {
  b = null;
 }
else {
  b = a.substring(1);
}
```
or, even better:
```(java)
String b;
if (a != null && a.length() >= 1) {
  b = a.substring(1);
 }
else {
  b = null;
}
```

Please remove all exaples like `? true : false; `  and use something real.

Reply:
Alright, will do these changes

Reply:
@pbludov But checkstyle isn't generating any violations in these cases.
As per the check, it only throws a violation for the ternary operator.

Reply:
The example should show the ternary operator as a violation, and if-else as a valid statement.
But the ternary operator should be something real, as the example in the description of the Check. Your example `boolean result = (x == 5) ? true : false;` is unnatural and does not tell the user how to avoid such violations.

Reply:
Got it 👍

Reply:
Done. Please check